pandas/tests/test_frame.py

```diff
@@ -8369,6 +8369,20 @@ def test_drop_duplicates(self):
         expected = df.iloc[[-2,-1]]
         assert_frame_equal(result, expected)
 
+        # GH 11376
+        df = pd.DataFrame([[1 , 0], [0, 2]])
+        assert_frame_equal(df.drop_duplicates(), df)
+
+        df = pd.DataFrame([[-2, 0], [0, -4]])
+        assert_frame_equal(df.drop_duplicates(), df)
+
```

Comment:
can u add the op example here as well
do we have sufficient coverage for various dtypes?